### PR TITLE
App directory accessibility review updates

### PR DESCRIPTION
**Pull request by LeKeve**
## Description

This PR fixes the first round of accessibility issues that came up during VSP staging review.

The following issues are resolved:

Headings increase by 1: [16437](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16437)
Accessible labels for "Find app" CTA: [16441](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16441)
Additional info toggle has aria-expanded attribute: [16464](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16464)
Scopes list should be an unordered list: [16466](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16466)
IE11 should render icons correctly: [16703](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16703)

Epic: https://app.zenhub.com/workspaces/vsp---product-support-5f85b91c14d8df0018fac414/issues/department-of-veterans-affairs/va.gov-team/16432

## Testing done

- local testing
- axe/coconut scans

## Acceptance criteria
- [x] axe scans return no violations

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/third-party-app-directory/sass/index.scss [hunk 1]
```diff
@@ -3,6 +3,13 @@
 }
 
 .third-party-app-directory {
+  h2 {
+    font-size: 1.17em;
```
Comment (rianfowler) [inline src/applications/third-party-app-directory/sass/index.scss:7]: might be able to use the `font-size` [utilities](https://design.va.gov/utilities/font-size)

File: src/applications/third-party-app-directory/sass/index.scss [hunk 2]
```diff
@@ -31,4 +38,8 @@
       margin-left: 86px;
     }
   }
+
+  button.va-button-link {
+    display: block;
```
Comment (rianfowler) [inline src/applications/third-party-app-directory/sass/index.scss:43]: might be able to use the `display` [utilities](https://design.va.gov/utilities/display)

**Review by rianfowler — APPROVED** [review]
Out of curiosity, what change fixed "IE11 should render icons correctly"?

I restarted the CI checks- they failed on a flakey unit test. If it continues to fail, I recommend merging / pushing the latest master.

**Comment by LeKeve** [comment]
> Out of curiosity, what change fixed "IE11 should render icons correctly"?
> 
> I restarted the CI checks- they failed on a flakey unit test. If it continues to fail, I recommend merging / pushing the latest master.

@rianfowler The new images hosted in s3 actually fixed the ie11 issue. 

I noticed that test failing a couple times. Merged master and can rerun if it continues failing.

**Pull request merged**